app/ante/eth.go

```diff
@@ -330,11 +322,12 @@ func (ctd CanTransferDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate
 			CoinBase:    common.Address{},
 			BaseFee:     baseFee,
 		}
-		evm := ctd.evmKeeper.NewEVM(coreMsg, cfg, evmtypes.NewNoOpTracer())
+		stateDB := statedb.New(ctx, ctd.evmKeeper, statedb.NewEmptyTxConfig(common.BytesToHash(ctx.HeaderHash().Bytes())))
```

Comment:
there's a few comments in `NewEmptyTxConfig` itself, it that good?
The empty config is good here because the `stateDB` is a temporary thing to run `evm.Context.CanTransfer`, the contents in it is not used.